packages/expo-updates/android/src/main/java/expo/modules/updates/manifest/EmbeddedManifest.java

```diff
@@ -0,0 +1,43 @@
+package expo.modules.updates.manifest;
+
+import android.content.Context;
+import android.util.Log;
+
+import androidx.annotation.Nullable;
+
+import org.apache.commons.io.IOUtils;
+import org.json.JSONObject;
+
+import java.io.InputStream;
+
+import expo.modules.updates.UpdatesConfiguration;
+import expo.modules.updates.loader.EmbeddedLoader;
+
+public class EmbeddedManifest {
```

Comment:
What is the purpose of having this be a class instead of a function? Is it just to cache the `sEmbeddedManifest`?